src/ensembl/src/content/app/browser/browser-tabs/BrowserTabs.tsx

```diff
@@ -23,23 +22,21 @@ const BrowserTabs: FunctionComponent<BrowserTabsProps> = (
       return;
     }
 
-    if (props.drawerOpened) {
+    if (props.isDrawerOpened) {
       props.toggleDrawer(false);
     }
 
     props.selectBrowserTabAndSave(value);
   };
 
   const getBrowserTabClasses = (trackType: TrackType) => {
-    const { activeGenomeId, drawerOpened, trackPanelModalOpened } = props;
-    const selectedBrowserTab =
-      props.selectedBrowserTab[activeGenomeId] || TrackType.GENOMIC;
+    const { isDrawerOpened, selectedBrowserTab, trackPanelModalOpened } = props;
     let classNames = styles.browserTab;
 
     if (
       props.ensObject.genome_id &&
       selectedBrowserTab === trackType &&
-      drawerOpened === false &&
+      isDrawerOpened === false &&
       trackPanelModalOpened === false
```

Comment:
!trackPanelModalOpened ?

Reply:
I'll also change the variable to `isTrackPanelModalOpened`